jenkins-ci/README.md

```diff
@@ -0,0 +1,75 @@
+# Jenkins CI
+Jenkins CI deploys OpenShift templates and verifies. This CI also runs functional tests from [jaeger-java-test](https://github.com/Hawkular-QE/jaeger-java-test) repository.
```

Comment:
@objectiser I have added on the [jaeger-java-test](https://github.com/Hawkular-QE/jaeger-java-test) repository README file.
Test cases are availabe on [test directory](https://github.com/Hawkular-QE/jaeger-java-test/tree/master/src/test/java/io/jaegertracing/qe/tests)